timelock-agent/src/main/java/com/palantir/timelock/invariants/TimeLockActivityCheckerFactory.java

```diff
@@ -50,7 +51,10 @@ public TimeLockActivityCheckerFactory(
 
     private ServiceCreator createServiceCreatorForRemote(String remoteUrl) {
         return ServiceCreator.withPayloadLimiter(
-                metricsManager, () -> getServerListConfig(remoteUrl), userAgent, () -> RemotingClientConfigs.DEFAULT);
+                metricsManager,
+                Refreshable.only(getServerListConfig(remoteUrl)),
```

Comment:
This is not actually refreshable, right? maybe we should leave a comment.

Reply:
Is `ClusterConfiguration` mutable? Otherwise it is not really a change

Reply:
I do not think so. It is a snapshot not an actual refreshable IIRC.